togglehq.gemspec

```diff
@@ -28,6 +28,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "simplecov"
 
   spec.add_dependency "faraday"
-  spec.add_dependency "net-http-persistent"
+  spec.add_dependency "net-http-persistent", "2.9.4"
```

Comment:
This is way too specific. We should allow the enclosing app define hard dependencies if they want to, but keep this version open ended.